pkg/ebpf/common.go

```diff
@@ -32,6 +32,9 @@ var (
 	ErrNotImplemented = errors.New("BPF-based system probe not implemented on non-linux systems")
 
 	nativeEndian binary.ByteOrder
+	// newAPIKernelVersion indicates the minimum version of kernel that uses the new API signature
+	// the affected APIs are kprobe/tcp_sendmsg, kprobe/udp_sendmsg, kprobe/udp_recvmsg
+	newAPIKernelVersion = "4.1.0"
```

Comment:
I don't like the word `new` here - will think on a better name. Lets make this `= stringToKernelCode("4.1.0")` though